gulpfile.js

```diff
@@ -59,7 +60,13 @@ gulp.task('tslint', function () {
 });
 
 function test() {
-    throw new Error('To run tests, open the extension in VS Code and hit F5 with the "Launch Tests" configuration. Alternatively, on Mac OS, you can run "npm test" on the command line.');
+    return gulp.src(['out/test/**/*.test.js', '!out/test/extension/**'])
```

Comment:
It is a gulp task, it's just not an anonymous function. As for why: That's how it was when I found it. I don't think there's a particular reason for it.